themekit.rb

```diff
@@ -1,9 +1,9 @@
 class Themekit < Formula
+  version 'v1.1.2'
   desc 'Theme Kit is a tool kit for manipulating shopify themes'
   homepage 'https://shopify.github.io/themekit/'
-  url 'https://shopify-themekit.s3.amazonaws.com/v1.1.1/darwin-amd64/theme'
-  sha256 '4b70165660e817e8d6a8f5f1ac8aca5f38ae9f77231f8ea53cc4f42700514a0a'
-  version 'v1.1.1'
+  url "https://shopify-themekit.s3.amazonaws.com/#{version}/darwin-amd64/theme"
```

Comment:
I only just discovered that we can use others keys here, e.g. https://github.com/Shopify/homebrew-shopify/blob/master/shopify-cli.rb